types/mux.d.ts

```diff
@@ -0,0 +1,250 @@
+// import { AxiosInstance } from 'axios';
+import { EventEmitter } from 'events';
+
+import { AssetMp4Support } from './interfaces/AssetMp4Support';
+import { AudioTrack } from './interfaces/AudioTrack';
+import { CreateAssetParams } from './interfaces/CreateAssetParams';
+import { InputInfo } from './interfaces/InputInfo';
+import { InputOverlaySettings } from './interfaces/InputOverlaySettings';
+import { InputTrack } from './interfaces/InputTrack';
+import { InputFile } from './interfaces/InputFile';
+import { InputSettings } from './interfaces/InputSettings';
+import { PlaybackId } from './interfaces/PlaybackId';
+import { PlaybackIdPolicy } from './interfaces/PlaybackIdPolicy';
+import { RequestOptions } from './interfaces/RequestOptions';
+import { StaticRenditions } from './interfaces/StaticRenditions';
+import { TextTrack } from './interfaces/TextTrack';
+import { VideoTrack } from './interfaces/VideoTrack';
+
+export declare class Base extends EventEmitter {
+  private _tokenId?: string;
+  private _tokenSecret?: string;
+  private _config?: RequestOptions;
+  private _requestOptions?: RequestOptions;
+
+  constructor(base: Base | RequestOptions);
+  constructor(tokenId: string, tokenSecret: string, config: RequestOptions);
+  constructor(
+    param?: Base | RequestOptions | string,
+    tokenSecret?: string,
+    config?: RequestOptions
+  );
+  constructor(...params: any[]);
+}
+
+declare type Track = VideoTrack | AudioTrack | TextTrack;
+
+type MasterAcces = 'none' | 'temporary';
```

Comment:
👀

Reply:
ty - I moved this type to it's own file and fixed the type